### PR TITLE
Revise api ref providers 2

### PR DESCRIPTION
**Pull request by DoguhanOzgurAkca**
**IMPORTANT: Please do not create a Pull Request without creating an issue first.**

_Any change needs to be discussed before proceeding. Failure to do so may result in the rejection of the pull request._

Please provide enough information so that others can review your pull request:

Explain the **details** for making this change. What existing problem does the pull request solve?

### Test plan (required)

Demonstrate the code is solid. If not, please add `WIP:` in its title.

### Closing issues

Put `closes #XXXX` in your comment to auto-close the issue that your PR fixes (if such).

### Self Check before Merge

Please check all items below before review.

-   [ ] Corresponding issues are created/updated or not needed
-   [ ] Docs are updated/provided or not needed
-   [ ] Examples are updated/provided or not needed
-   [ ] TypeScript definitions are updated/provided or not needed
-   [ ] Tests are updated/provided or not needed
-   [ ] Changesets are provided or not needed

### CODE REVIEW

File: documentation/docs/api-reference/core/providers/i18n-provider.md [hunk 1]
```diff
@@ -128,7 +125,7 @@ i18n.use(Backend)
 export default i18n;
 ```
 
-### Wraping app with React.Suspense
+### Wraping the app with React.Suspense
```
Comment (necatiozmen) [inline documentation/docs/api-reference/core/providers/i18n-provider.md:128]:
```suggestion
### Wrapping the app with React.Suspense
```

File: documentation/docs/api-reference/core/providers/i18n-provider.md [hunk 2]
```diff
@@ -45,30 +45,27 @@ const App: React.FC = () => {
 };
 ```
 
-`i18nProvider` allows us to put translation features to the followings hooks:
+## Example
 
--   [`useTranslate`][use-translate] shows translation between different languages.
```
Comment (necatiozmen) [inline documentation/docs/api-reference/core/providers/i18n-provider.md:50]: Are these gone?

File: documentation/docs/api-reference/core/providers/live-provider.md [hunk 3]
```diff
@@ -22,11 +22,13 @@ const liveProvider = {
...
 
+---
+
 :::tip
-**refine** includes out-of-the-box live providers to use in your projects like:
+**refine** includes some out-of-the-box live providers to use in your projects such as:
 
 -   **Ably** &#8594 [Source Code](https://github.com/refinedev/refine/blob/master/packages/ably/src/index.ts) - [Demo](https://codesandbox.io/embed/github/refinedev/refine/tree/next/examples/live-provider-ably/?view=preview&theme=dark&codemirror=1)
--   **Supabase** &#8594 [Source Code](https://github.com/refinedev/refine/blob/master/packages/supabase/src/index.ts#L187)
+-   **supabase** &#8594 [Source Code](https://github.com/refinedev/refine/blob/master/packages/supabase/src/index.ts#L187)
```
Comment (necatiozmen) [inline documentation/docs/api-reference/core/providers/live-provider.md:31]: Keep it uppercase please

File: documentation/docs/api-reference/core/providers/i18n-provider.md [hunk 4]
```diff
@@ -747,20 +748,16 @@ Finally, we will create the `<PostList>` page and then we will translate texts u
...
-    useTable,
-    TextField,
     EditButton,
+    List,
     ShowButton,
+    TextField,
+    useTable,
 } from "@refinedev/antd";
-import { Table, Space } from "antd";
+import { useMany, useTranslate } from "@refinedev/core";
```
Comment (necatiozmen) [inline documentation/docs/api-reference/core/providers/i18n-provider.md:757]: Keep the //highlight-next-line please

File: documentation/docs/api-reference/core/providers/i18n-provider.md (hunk 2 above)
Comment (DoguhanOzgurAkca) [reply documentation/docs/api-reference/core/providers/i18n-provider.md:50]: added back in

File: documentation/docs/api-reference/core/providers/i18n-provider.md (hunk 4 above)
Comment (DoguhanOzgurAkca) [reply documentation/docs/api-reference/core/providers/i18n-provider.md:757]: eslint probably made it, reverted this part fully.

**Review by necatiozmen — APPROVED** [review]

**Pull request merged**